### PR TITLE
Improve mobile style

### PR DESCRIPTION
**Pull request by sandrica89**
* Use design system typography
* Improve spacing on mobile

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -41,6 +41,11 @@
   padding-bottom: 0 !important; /* overwrite DSFR default */
 }
 
+.fr-text--lead,
+.fr-text--lg {
+  margin-bottom: 2rem; /* overwrite DSFR default */
```
Comment (sandrica89) [inline frontend/src/App.css:46]: DSFR margin bottom are really high (3rem for lead and 2.5rem for lg) it doesn't look great on our website

File: frontend/src/components/Landing/Landing.jsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const Landing = () => {
       </div>
       <div className={styles.sectionLight}>
         <Section
-          title="Vous êtes psychologue ?"
+          title="Vous êtes psychologue ?"
```
Comment (sandrica89) [inline frontend/src/components/Landing/Landing.jsx:25]: Unbreakable space added to avid having question mark alone in a second line on mobile

File: frontend/src/components/Page/UnderlinedTitle.jsx [hunk 3]
```diff
@@ -5,10 +5,11 @@ import styles from './underlinedTitle.cssmodule.scss';
 
 const UnderlinedTitle = ({ title, big, className, backgroundColor }) => {
   const style = big ? 'bigTitle' : 'title';
+  const titleStyle = big ? '' : 'fr-h2';
 
   return (
     <div className={className}>
-      <h1 className={classNames(styles[style], styles[`${style}-${backgroundColor}`])}>
+      <h1 className={classNames(titleStyle, styles[style], styles[`${style}-${backgroundColor}`])}>
```
Comment (desoindx) [inline frontend/src/components/Page/UnderlinedTitle.jsx:12]: you could use this syntaxe:
```suggestion
      <h1 className={classNames(styles[style], styles[`${style}-${backgroundColor}`], {'fr-h2': big})}>
```

**Comment by desoindx** [comment]
Note que la FAQ n'est plus ok en mobile (mais c'est pas lié a cette PR) j'ai crée un trello 
https://trello.com/c/tHMhvczR/249-fix-faq-display-on-mobile-view

**Review by desoindx — APPROVED** [review]

**Review by desoindx — APPROVED** [review]

**Pull request merged**